### ISSUE TITLE
The controls for master - admission control showing wrong status

### ISSUE THREAD

**Issue by junaid18183** [aquasecurity/kube-bench#49]
For master , controls 1.1.11,1.1.12,1.1.13,1.1.14,1.1.15,1.1.25,1.1.28,1.1.33  showing wrong status.

example - 

```
 # ./kube-bench master -c 1.1.11,1.1.12,1.1.13,1.1.14,1.1.15,1.1.25,1.1.28,1.1.33
Unexpected Client version 1.6
Unexpected Server version 1.6
[INFO] 1 Master Node Security Configuration
[INFO] 1.1 API Server
[PASS] 1.1.11 Ensure that the admission control policy is not set to AlwaysAdmit (Scored)
[FAIL] 1.1.12 Ensure that the admission control policy is set to AlwaysPullImages (Scored)
[FAIL] 1.1.13 Ensure that the admission control policy is set to DenyEscalatingExec (Scored)
[FAIL] 1.1.14 Ensure that the admission control policy is set to SecurityContextDeny (Scored)
[PASS] 1.1.15 Ensure that the admission control policy is set to NamespaceLifecycle (Scored)
[FAIL] 1.1.25 Ensure that the admission control policy is set to PodSecurityPolicy (Scored)
[FAIL] 1.1.28 Ensure that the admission control policy is set to ServiceAccount (Scored)
[FAIL] 1.1.33 Ensure that the admission control policy is set to NodeRestriction (Scored)

== Remediations ==
1.1.12 Edit the /etc/kubernetes/apiserver file on the master node and set the KUBE_ADMISSION_CONTROL parameter to "--admission-control=...,AlwaysPullImages,..."
1.1.13 Edit the /etc/kubernetes/apiserver file on the master node and set the KUBE_ADMISSION_CONTROL parameter to "--admission-control=...,DenyEscalatingExec,..."
1.1.14 Edit the /etc/kubernetes/apiserver file on the master node and set the KUBE_ADMISSION_CONTROL parameter to "--admission-control=...,SecurityContextDeny,..."
1.1.25 Follow the documentation and create Pod Security Policy objects as per your environment. Then, edit the /etc/kubernetes/apiserver file on the master node and set the KUBE_ADMISSION_CONTROL parameter to "--admission-control=...,PodSecurityPolicy,..."
1.1.28 Follow the documentation and create ServiceAccount objects as per your environment. Then, edit the /etc/kubernetes/apiserver file on the master node and set the KUBE_ADMISSION_CONTROL parameter to "--admissioncontrol=...,ServiceAccount,..."
1.1.33 Follow the Kubernetes documentation and configure NodeRestriction plug-in on kubelets. Then, edit the /etc/kubernetes/apiserver file on the master node and set the KUBE_ADMISSION_CONTROL parameter to "--admissioncontrol=...,NodeRestriction,..."

== Summary ==
2 checks PASS
6 checks FAIL
0 checks WARN
```
Command output - 

```
 # systemctl status kube-apiserver -l
● kube-apiserver.service - Kubernetes API Server
   Loaded: loaded (/usr/lib/systemd/system/kube-apiserver.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2017-09-12 03:21:32 EDT; 56min ago
     Docs: https://github.com/GoogleCloudPlatform/kubernetes
 Main PID: 2733 (hyperkube)
    Tasks: 21
   Memory: 134.6M
      CPU: 42.538s
   CGroup: /system.slice/kube-apiserver.service
           └─2733 /usr/bin/hyperkube apiserver --logtostderr=true --v=0 --etcd-servers=https://<MASKED_IP>:2379 --insecure-bind-address=127.0.0.1 --allow-privileged=true --service-cluster-ip-range=10.3.0.0/24 --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota,AlwaysPullImages,DenyEscalatingExec,SecurityContextDeny,PodSecurityPolicy --anonymous-auth=false --profiling=false --repair-malformed-updates=false --advertise-address=10.88.51.70 --bind-address=0.0.0.0 --etcd-cafile=/etc/kubernetes/ssl/ca.pem --etcd-certfile=/etc/kubernetes/ssl/client.pem --etcd-keyfile=/etc/kubernetes/ssl/client-key.pem --runtime-config=extensions/v1beta1=true,extensions/v1beta1/networkpolicies=true,batch/v2alpha1 --secure-port=4443 --tls-cert-file=/etc/kubernetes/ssl/apiserver.pem --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem --client-ca-file=/etc/kubernetes/ssl/ca.pem --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem

```

OS and kube version : 

```
vm70:/home/junedm/kube-bench # kubectl version
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.1", GitCommit:"$Format:%H$", GitTreeState:"not a git tree", BuildDate:"2017-07-23T22:07:37Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.1", GitCommit:"$Format:%H$", GitTreeState:"not a git tree", BuildDate:"2017-07-23T22:07:37Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
vm70:/home/junedm/kube-bench # uname -a
Linux vm70 4.4.21-69-default #1 SMP Tue Oct 25 10:58:20 UTC 2016 (9464f67) x86_64 x86_64 x86_64 GNU/Linux
vm70:/home/junedm/kube-bench # lsb_release -a
LSB Version:    n/a
Distributor ID: SUSE
Description:    SUSE Linux Enterprise Server 12 SP2
Release:        12.2
Codename:       n/a
vm70:/home/junedm/kube-bench #
```

**Comment by junaid18183** [comment]
Only the first policy after -  --admission-control is getting considered and Passing , rest all are marked as FAIL

**Comment by junaid18183** [comment]
Have created a PR please check

**Comment by lizrice** [comment]
Fixed by #50 - thanks so much @junaid18183!
